fmriprep/workflows/bold/hmc.py

```diff
@@ -10,16 +10,16 @@
 """
 
 from nipype.pipeline import engine as pe
-from nipype.interfaces import utility as niu, fsl
+from nipype.interfaces import utility as niu, afni, fsl
 from niworkflows.interfaces import NormalizeMotionParams
+from ...interfaces import Volreg2ITK, MCFLIRT2ITK
 from ...engine import Workflow
-from ...interfaces import MCFLIRT2ITK
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
 
 # pylint: disable=R0914
-def init_bold_hmc_wf(mem_gb, omp_nthreads, name='bold_hmc_wf'):
+def init_bold_hmc_wf(use_mcflirt, mem_gb, omp_nthreads, name='bold_hmc_wf'):
```

Comment:
If we eventually want to be able to swap in any MC algorithm, `use_mcflirt` is not a great name for the parameter. What about `motion_correction` that takes a string such as `'3dvolreg'` or `'mcflirt'` (and I would accept `'afni'` and `'fsl'` as synonyms, at least at the command line).